src/VirtoCommerce.Platform.Web/Controllers/Api/AssetsController.cs

```diff
@@ -186,6 +186,9 @@ public async Task<ActionResult<BlobInfo[]>> UploadAssetAsync([FromQuery] string
         [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
         public async Task<ActionResult> DeleteBlobsAsync([FromQuery] string[] urls)
         {
+            if (urls.IsNullOrEmpty())
+                return BadRequest("Please, specify all deleting assets urls.");
```

Comment:
I think it's better to say:
"Please, specify at least one asset URL to delete."